### PR TITLE
Finish_mobile_first

### PR DESCRIPTION
**Pull request by MILINDI7**
## What?
Adding the works section and the about section in the personal portfolio project.

## Why?
As the project requirement, adding where recent work can be published and more about you is a best step on a professional personal portfolio.

## How?
By the use of HTML semantics and CSS styling, I added the required sections and pushed them to github, waiting to be reviewed for better performance.

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @MILINDI7, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not follow GitHub flow correctly. Please your branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section' or 'work-section'). Kindly create a new branch and copy the changes made on this branch into your new branch. Then you open a pull request from the new branch and request another review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by MILINDI7** [comment]
Hello!

I see that the pull request reveals every detail about the project changes
and commits , and the branch should not be a long text, that's why I named
my branch feature as a remote branch and explained every detail in the pull
request.

Thank you!

**Comment by Kamba56** [comment]
> Only

- [ ] Please check [this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md#feature-branches) for more details.

File: index.css [hunk 1]
```diff
@@ -64,3 +64,231 @@ p {
...
+
+.more p {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: center;
+  color: #f5f5ff;
+}
```
Comment (Abenezer-Tilahun) [inline index.css:294]: - Please try to  align the elements according to the requirement, you should use Flex-box in all 2 sections(work and about section.) For Minimum mobile screen size at  375px, it's not aligned according to requirement(checkout below image of your project)
- Both work and about section must have the margins (top, bottom, left, right) defined in the Template design.
-  You didn't follow the Figma design Requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this :
you
ypur cont
 
The Figma design Look likes : 

fig co

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MILINDI7 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README 

almost

## Required Changes ♻️
_Check the comments below the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 2]
```diff
@@ -64,3 +64,230 @@ p {
...
+.esentials ul li:nth-child(4) {
+  border-left: 1px solid #555;
+}
+
+button {
+  background-color: #7d73fc;
+  width: 90px;
+  height: 48px;
+  padding: 8px;
+}
```
Comment (Kamba56) [inline index.css:262]: - [ ] As requested by the previous reviewer, please make your buttons hover and pressed state look like the design template. You can use the `:hover` pseudo-class for the hover state and the `:active` pseudo-class for the pressed state
185765487-6af519e8-dfd5-4272-b2d3-b95e5bf244f9.

- [ ] As requested by the previous reviewer, please make all your alignments and spacing look like the design template.
*Your design*
185765855-419d15ed-a204-4f8c-a417-5d94a3440fe4

*Figma template*
Screenshot 2022-08-20 230128
.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @MILINDI7, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [x] All linters checks passed


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @MILINDI7 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**